### DISCUSSION TITLE
Select tests via a test plan file | allure-jest

### DISCUSSION THREAD

**Discussion by yaroslavchaadaev** [allure-framework/allure2#2311] [Questions & Support]
Hey!

Could you tell me, when will allure-jest add the ability to selectively rerun tests?

Your documentation says that this feature already exists, but technical support said that this functionality has not yet been implemented.

https://allurereport.org/docs/jest/#select-tests-via-a-test-plan-file

**Comment by cheshi-mantu** [comment]
Genearlly, documentation at https://allurereport.org/docs/jest/#select-tests-via-a-test-plan-file has the most recent updates, still you need to make sure you are using the most recent integration in your project.

If the feature is not working for you, you can [start a new discussion](https://github.com/orgs/allure-framework/discussions) to get help or [file a bug report](https://github.com/allure-framework/allure-js/issues) is you are 100% sure the feature is not working accordingly to the documentation.

For this particular case, the selective test run is not implemented for Jest as of yet and this is work in progress.
